templates/readme.ts

```diff
@@ -0,0 +1,80 @@
+import type { ReadmeTemplate } from "../generator/interfaces";
+
+export default (r: ReadmeTemplate): string => {
+  return `
```

Comment:
Huh? Can this not just load the file or not exist? This will go out of date for sure?

Reply:
resolved elsewhere.